Framework/script/RepoCleaner/o2-qc-repo-delete-runs

```diff
@@ -0,0 +1,126 @@
+    parser.add_argument('--dry-run', action='store_true',
+                        help='Dry run, no actual deletion nor modification to the CCDB.')
+    parser.add_argument('--path', dest='path', action='store', default="",
+                        help='Clean this path.', required=True)
+    parser.add_argument('--runs-csv-file', dest='runs_file', action='store', help='A csv file with a header and the '
+                                                                                  'column `runNumber` contains the run',
+                        required=True)
+    parser.add_argument('--one-by-one', action='store_true', help='Ask confirmation for each deletion')
+    parser.add_argument('--print-list', action='store_true', help='Only print the list of objects that would be deleted')
+    args = parser.parse_args()
+    dryable.set(args.dry_run)
+    logging.info(args)
+    return args
+
+
+def prepare_main_logger():
+    logger = logging.getLogger()
+    # Logging (split between stderr and stdout)
+    formatter = logging.Formatter(fmt='%(asctime)s - %(levelname)s - %(message)s', datefmt='%d-%b-%y %H:%M:%S')
+    h1 = logging.StreamHandler(sys.stdout)
+    h1.setLevel(logging.DEBUG)
+    h1.addFilter(lambda record: record.levelno <= logging.INFO) # filter out everything that is above INFO level
+    h1.setFormatter(formatter)
+    logger.addHandler(h1)
+    h2 = logging.StreamHandler(sys.stderr)
+    h2.setLevel(logging.WARNING)     # take only warnings and error logs
+    h2.setFormatter(formatter)
+    logger.addHandler(h2)
+
+
+def run(args):
+    ccdb = Ccdb(args.url)
+
+    total_deleted = 0
+
+    file = open(args.runs_file)
+    csvreader = csv.DictReader(file)
+    for row in csvreader:
+        run_number = row["runNumber"]
+        logging.debug(f"run : {run_number}")
+
+        versions = ccdb.getVersionsList(args.path, "", "", run_number)
+        logging.info("Here are the objects that are going to be deleted: ")
+        for v in versions:
+            logging.info(v)
+        logging.info(f"Number of items: {len(versions)}")
+
+        if args.print_list:
+            continue
+
+        logging.warning("****** ARE YOU ABSOLUTELY SURE YOU WANT TO CONTINUE ? ******")
+        answer = input("Yes/No \n  ")
+        if answer.lower() not in ["y", "yes"]:
+            exit(0)
+
+        for v in versions:
+            logging.info(f"Ready to delete {v}")
+            if args.one_by_one:
+                answer = input("  Continue? y/n\n  ")
+                if answer.lower() in ["y", "yes"]:
+                    ccdb.deleteVersion(v)
+                    logging.debug("")
+                elif answer.lower() in ["n", "no"]:
+                    logging.info("   skipping")
+                else:
+                    logging.error("   wrong input, skipping")
+            else:
+                ccdb.deleteVersion(v)
+                logging.debug("")
+
+        logging.info(f"Deleted items: {len(versions)}")
```

Comment:
the count will also include the versions where you said "no". That could be misleading and scare us in the future.